pkgs/applications/window-managers/cagebreak/default.nix

```
@@ -0,0 +1,54 @@
+{ stdenv, fetchFromGitHub
+, meson, ninja, pkg-config, wayland, scdoc, makeWrapper
+, wlroots, wayland-protocols, pixman, libxkbcommon
+, cairo , pango, fontconfig, pandoc, udev
+, xwayland ? null
+, nixosTests
+}:
+
+stdenv.mkDerivation rec {
+  pname = "cagebreak";
+  version = "1.4.2";
+
+  src = fetchFromGitHub {
+    owner = "project-repo";
+    repo = "cagebreak";
+    rev = version;
+    sha256 = "+Ww1rsUR7qe/BixLPR8GiRc3C6QmpLzWpT2wym8b4/M=";
+  };
+
+  nativeBuildInputs = [ meson ninja pkg-config wayland scdoc makeWrapper ];
+
+  buildInputs = [
+    wlroots wayland wayland-protocols pixman libxkbcommon cairo
+    pango.dev fontconfig pandoc udev
```

```suggestion
    pango.dev fontconfig pandoc systemd
```

that is an alias IIRC

---

Fixed